packages/database/src/migrations/20210322030033-FixStriveAfterTupaiaServiceConversion-modifies-data.js

```diff
@@ -14,7 +14,17 @@ exports.setup = function (options, seedLink) {
   seed = seedLink;
 };
 
-const cleanDhisMetadataForStriveCases = async db => {
+const deleteDhisSyncTableRecordsForStriveCases = async (db, table) =>
+  db.runSql(`
+    DELETE FROM ${table}
+    USING survey_response sr
+    JOIN survey s on s.id = sr.survey_id
+    WHERE
+      sr.entity_id = ${table}.record_id AND
+      s.code = 'SCRF';
+`);
```

Comment:
Sorry if I'm being a bit pedantic, but I think not all survey responses are attached to case entity in `SCRF` survey (not sure why, but I think it could be a mistake somehow)...? 

When I run this query (converted from your delete statement):
```
SELECT distinct e.type from dhis_sync_log d
JOIN survey_response sr on sr.entity_id = d.record_id
JOIN survey s on s.id = sr.survey_id
JOIN entity e on e.id = sr.entity_id
WHERE s.code = 'SCRF'
```

It gives the result (expected only case, but not sure how a facility slipped through):
![image](https://user-images.githubusercontent.com/63621318/113954733-c0dcac00-985d-11eb-85b7-bb1d4527cac6.png)

Tracing the facility entity sync records, it is only 1 facility:
![facility](https://user-images.githubusercontent.com/63621318/113954889-131dcd00-985e-11eb-94ac-5e705bf2794d.PNG)

So I guess we don't want to delete the sync record of the facility entities If I'm correct... If so we can add a where clause `e.type = 'case'` to the delete statement?

Reply:
Not pedantic at all, on the contrary thanks for giving this a thorough look! Yeah not sure why a `facility` slipped through, but it's good to keep it safe and add the extra condition you mentioned